pkg/apis/istio/v1alpha2/values/fixup_structs/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+const (
+	replaceStructToken = "// GOSTRUCT: "
+)
+
+func usage() {
```

Comment:
yep I am just not sure whether it is common to print usage information like this for internal build tool. Might be better to update this information to the README, I remember I added a README here for patch instruction before.

Reply:
I see, yes, that's a fair point. I've removed that and also added some overview in the code. Note that I changed the approach because using the annotations I had before did not fully work, since method calls were still using the wrong types.